velox/functions/sparksql/Hash.cpp

```diff
@@ -175,8 +339,23 @@ std::vector<std::shared_ptr<exec::FunctionSignature>> hashSignatures() {
 std::shared_ptr<exec::VectorFunction> makeHash(
     const std::string& name,
     const std::vector<exec::VectorFunctionArg>& inputArgs) {
-  static const auto kHashFunction = std::make_shared<HashFunction>();
+  static const auto kHashFunction = std::make_shared<Murmur3HashFunction>();
   return kHashFunction;
 }
 
+std::vector<std::shared_ptr<exec::FunctionSignature>> xxhash64Signatures() {
+  return {exec::FunctionSignatureBuilder()
+              .returnType("bigint")
+              .argumentType("any")
```

Comment:
This signature doesn't seem quite right. It suggests the function supports all types, but it looks like it doesn't support arrays, maps, structs, dates, timestamps, etc.

Reply:
I try to specify the accurate function signature, but I don't know how to represent such case, do you have more suggested solution? Maybe like that https://github.com/facebookincubator/velox/commit/b5ab3be8960aaf98154c725b3f3c10ae9c8207a4 ?
```
Reason: Scalar function signature is not supported: xxhash64(VARCHAR, INTEGER).
```

Reply:
I add the comment above function signature to describe supported types, does it work?